feat(Intents): add "Start With JIT" intent

junepark678 opened the pull request:
### Changes

<!-- Fill this list with what your PR changes. Example: -->
- Repurpose "View App" intent to start an app with JIT enabled

<!-- If your PR doesn't close an issue, you can remove the next line. -->
Closes #166 #443

rileytestut committed:
Adds “Clear Cache” button to remove temporary files and uninstalled app backups

(cherry picked from commit 3adfc9db6de2f8e05b1d7dee1bbe486493c961c0)

rileytestut committed:
Adds “Clear Cache” description to Techy Things section footer

(cherry picked from commit 913db5131be741a8d2f498b85a426ac2e43cf8c5)

rileytestut committed:
Fixes AppViewController deprecation warnings

rileytestut committed:
Fixes “variable mutated after capture by sendable closure” warning

rileytestut committed:
Fixes UIDocumentPickerViewController deprecation warnings

rileytestut committed:
Fixes incorrect cell height for some News items

We need to take layoutMargins into account when calculating the height of the prototype cell.

rileytestut committed:
Fixes CollapsingTextView “TextKit 1 compatibility mode” runtime warning

rileytestut committed:
Enforces 77x31 minimum size for PillButton

amsam0 committed:
chore: Remove old apps.json/app.json files

amsam0 committed:
[skip ci] chore: rename tempEnt.plist to ReleaseEntitlements.plist to reduce future confusion

amsam0 committed:
Revert "Don't reinstall on first SideStore refresh"

This reverts commit 40e1225b875922d8d0dfe0e713434e4150a26e03.

amsam0 committed:
fix: always reinstall when refreshing ourselves

amsam0 committed:
fix: add .AltWidget to app group ID when modifying for SideStore

amsam0 committed:
fix: hopefully reduce ApplicationVerificationFailed errors by removing _CodeSignature folders since those may cause a problem

amsam0 committed:
fix: force close SideStore after 3 seconds if still reinstalling

amsam0 committed:
[skip ci] ci: fully rename SideStore.ipa, even after extracting the artifact zip

amsam0 committed:
fix: removing _CodeSignature folder before resigning

amsam0 committed:
fix: use a notification instead of an alert for force close

amsam0 committed:
fix: ensure minimuxer is started when refreshing in the background

amsam0 committed:
fix: open Safari instead of force closing and provide a fallback for users with notifications disabled

rileytestut committed:
Fixes updating apps with manually-removed app extensions (e.g. uYou+)

SoY0ung committed:
Fix 'The name for this app is invalid' error
This error is related to App ID creation failure.
App ID name must be an ascii text. It is not allowed to create an App ID with non-ascii text like Chinese, Japanese.
If the name is NOT an ascii text, using bundleID instead.

SoY0ung committed:
Optimizing function calls
Thanks for @ktprograms advice

SoY0ung committed:
Fix 'The name for this app is invalid' error(#361)

Fix 'The name for this app is invalid' error when sideloading with non-ascii name ipa

amsam0 committed:
Anisette V3 (#324)

* initial anisette V3 implementation

* update V3 urls and log version

* fix crash where FetchAnisetteDataOperation.clientInfo would be nil when getting anisette V3 without provisioning first

* move adi.pb reset to its own button instead of doing it on sign out

* fallback to V1 if client_info fails

* make sure to unwrap optional strings

* feat(anisette): update v3 usage, improve error messages and names, report v3 errors to the user

* refactor(anisette): reduce duplicate JSON to anisette code

* fixes(anisette v3): improve errors, fix v3 server check, fix some edge cases where SideStore could crash and instead return an error, retry on -45061

lonkelle committed:
Update README.md

Signed-off-by: Joelle Stickney <joellestickney+commit@gmail.com>

lonkelle committed:
Update README.md

Co-authored-by: JoeMatt <mail@joemattiello.com>

Signed-off-by: Joelle Stickney <joellestickney+commit@gmail.com>

lonkelle committed:
Update README.md

Co-authored-by: Joe Mattiello <mail@joemattiello.com>

Signed-off-by: Joelle Stickney <joellestickney+commit@gmail.com>

lonkelle committed:
Update README.md

Co-authored-by: Joe Mattiello <mail@joemattiello.com>

lonkelle committed:
Co-authoring all the things

Co-authored-by: Joe Mattiello <mail@joemattiello.com>

lonkelle committed:
Merge pull request #365 from lonkelle/develop

Co-authored-by: Joe Mattiello <mail@joemattiello.com>

Spidy123222 committed:
Change default anisette to v3 anisette server (#367)

Spidy123222 committed:
Change label to 0.4.0

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

Spidy123222 committed:
Upload config (#383)

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

Spidy123222 committed:
Add EmuPalace Source

SternXD committed:
Update pr.yml

Signed-off-by: Stern <xsternent@gmail.com>

SternXD committed:
Update pr.yml

Signed-off-by: Stern <xsternent@gmail.com>

bogotesr committed:
Change Patreon screen again

Refresh to "support us" and include social media.

Spidy123222 committed:
Replace old servers with macley v3 ansiette

nythepegasus committed:
Fix building on Xcode 15

nythepegasus committed:
Fix libfragmentzip's search path

nythepegasus committed:
Update AltSign and OpenSSL packages

bogotesr committed:
Merge pull request #387 from bogotesr/develop

Fix up patreon screen and add socials

nythepegasus committed:
Update Macley omnisette server IP

Signed-off-by: Nythepegasus <nythepegasus84@gmail.com>

Spidy123222 committed:
[skip ci] Add Taurine

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

Spidy123222 committed:
[skip ci] use odyssey team unified source

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

nythepegasus committed:
These vars don't change, let's use let keyword

nythepegasus committed:
Xcode 15 keeps adding imobiledevice.swift

nythepegasus committed:
Add various retries to the minimuxer calls

nythepegasus committed:
Go to home screen instead of Safari/notif combo

nythepegasus committed:
Revert "Fix building on Xcode 15"

This reverts commit 3de24dcfce98014d3710b49eec2f770b7d1be9fa.

polymo1 committed:
[skip ci] em_proxy link update (#412)

Signed-off-by: Wes Bryie <wes@wesbryie.tech>

nythepegasus committed:
Fix Actions builds for AltWidget

nythepegasus committed:
Update AltWidget/ReleaseEntitlements.plist

Signed-off-by: Nythepegasus <me@nythepegas.us>

nythepegasus committed:
Move the first app group to be known shared group

nythepegasus committed:
Update AltWidget assets

nythepegasus committed:
Remove unused Riley and Shane images from bundle

nythepegasus committed:
Revert the retries here as these seem buggier

nythepegasus committed:
Merge branch 'develop' into patch/fix-widget

nythepegasus committed:
Finally fix retries for minimuxer calls

nythepegasus committed:
Change the lock screen icon to be legible

Signed-off-by: nythepegasus <me@nythepegas.us>

nythepegasus committed:
Change circle and logo to just be circle for now

nythepegasus committed:
Remove extra returns and make sure to decrement

nythepegasus committed:
Reintroduce notification/pop-up

nythepegasus committed:
Fix going to the home screen

nythepegasus committed:
Merge pull request #414 from SideStore/patch/fix-widget

[Patch] Change Widget entitlements to search hardcoded group first

nythepegasus committed:
[skip ci] Merge pull request #419 from Nythepegasus/chore/up-fixes

[Chore] Pull upstream changes from AltStore

nythepegasus committed:
Update default anisette server

nythepegasus committed:
Bump version to 0.5.0

Spidy123222 committed:
Make Notification explanation smaller for refresh

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

Spidy123222 committed:
Fix message and put in proper spot.

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

nythepegasus committed:
[skip ci] Change Discord custom invite to static website invite

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

lonkelle committed:
Added Quantum Source to trusted sources

Signed-off-by: Joelle Stickney <joellestickney+commit@gmail.com>

lonkelle committed:
Removed Quantum Source

Signed-off-by: Joelle Stickney <joellestickney+commit@gmail.com>

Spidy123222 committed:
Add ignited source

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

nythepegasus committed:
Update emuplace source

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

Spidy123222 committed:
hopefully fix error code build error

Spidy123222 committed:
please o riley build

Spidy123222 committed:
get rest of batcherror

Spidy123222 committed:
fix the mighty error

Spidy123222 committed:
attempt fix settings

Spidy123222 committed:
please fix to show button

Spidy123222 committed:
Add button to storyboard 🙄

Spidy123222 committed:
change order of settings debug section

Spidy123222 committed:
make button function again

Spidy123222 committed:
move debug row

Spidy123222 committed:
Update Altsign (August 6) (#467)

This changes the git commit or be our latest altsign version.

amsam0 committed:
[skip ci] Add more information to staging errors (#468)

* Point to my forks and attempt to add more information to staging errors

* Improve error message a bit

* Revert fetch-prebuilt.sh changes

* Undo some whitespace changes

* missed one

* oops

* [skip ci]

* [skip ci]

* [skip ci] remove staging directory from install app error

Signed-off-by: naturecodevoid <44983869+naturecodevoid@users.noreply.github.com>

---------

Signed-off-by: naturecodevoid <44983869+naturecodevoid@users.noreply.github.com>
Co-authored-by: Dadoum <dadoum@protonmail.com>

amsam0 committed:
Update Swift Packages and submodules (#469)

* Update Swift Packages

* Update submodules

* make sure it builds

---------

Signed-off-by: naturecodevoid <44983869+naturecodevoid@users.noreply.github.com>

Spidy123222 committed:
Change version to 0.5.2

amsam0 committed:
0.5.1

amsam0 committed:
Build nightly with latest minimuxer changes to attempt to fix plist_from_memory crash

amsam0 committed:
Bump version to 0.5.2 so nightly builds have a higher SemVer version than stable

bogsen committed:
Use provisioning profile details instead of guessing active app limit

nythepegasus committed:
Remove buggy retry code finally

nythepegasus committed:
Change this to be hardcoded SideStore search

Spidy123222 committed:
[skip ci] Remove emuthreeds from trusted sources

nythepegasus committed:
Add newly compiled AltBackup

nythepegasus committed:
Revert this change

nythepegasus committed:
Hardcode SideStore's URL scheme for now

nythepegasus committed:
Add iOS 17 JIT error notice with other errors

nythepegasus committed:
Update limd/libplist submodules

nythepegasus committed:
Attempt to fix submodule dependencies for GH runner

nythepegasus committed:
Update Roxas

nythepegasus committed:
[skip ci] Bump nightly to 0.5.3

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

amsam0 committed:
Use forked libplist

amsam0 committed:
Revert "Attempt to fix submodule dependencies for GH runner"

This reverts commit 5cff914ff35652d245688d7e90af6322b548bd65.

amsam0 committed:
Revert most of xcodeproj changes

nythepegasus committed:
Fix AltWidget App Group issue

Spidy123222 committed:
Change pairing file link with new wiki link

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

nythepegasus committed:
[skip ci] Update Ignited source URL

junepark678 committed:
feat(App IDs): make expiration dates more specific

junepark678 committed:
chore(App IDs): localize Unknown string

junepark678 committed:
feat(My Apps): make expiration dates more specific

junepark678 committed:
bugfix(App IDs, My Apps): calculate in correct direction in time (we aren't time travelers)

junepark678 committed:
bugfix(App IDs, My Apps): display only necessary information

junepark678 committed:
bugfix(App IDs, My Apps): fix date display

junepark678 committed:
bugfix(settings): fix rounding issues on clear cache button (#536)

junepark678 committed:
chore(App IDs, My Apps): change to use `DateComponentsFormatter.UnitsStyle.abbreviated`

junepark678 committed:
chore(App IDs, My Apps): change back to full

junepark678 committed:
feat: remove reliance on Info.plist for getting udid

junepark678 committed:
feat(Intents): add "Start With JIT" intent

junepark678 committed:
feat(Operations): don't idle timeout during installations

junepark678 committed:
chore(Clear Cache): do proper error handling

nythepegasus committed:
[skip ci] Add other project maintainers to CODEOWNERS

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

SternXD committed:
[skip ci] ci: add workflow_dispatch

Signed-off-by: Stern <xsternent@gmail.com>

nythepegasus committed:
[skip ci] Bump nightly to 0.5.4

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

junepark678 committed:
bugfix: make it able to be toggled, fix bug in crash on installation

junepark678 committed:
Merge branch 'develop' into NoIdle

Signed-off-by: June Park <rjp2030@outlook.com>

junepark678 committed:
bugfix: fix appending to a list that is nil

SternXD committed:
Merge pull request #539 from junepark678/NoIdle

feat(Operations): don't idle timeout during installations

junepark678 committed:
bugfix: fix removal of attributes

junepark678 committed:
[skip ci] bump nightly to 0.5.5

Signed-off-by: June Park <rjp2030@outlook.com>

junepark678 committed:
bugfix(NoIdle): Fix slider not being set to correct value on load

Spidy123222 committed:
add description on what disable idle timeout toggle

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

Spidy123222 committed:
Change button text for adding Siri to refresh apps

Signed-off-by: Spidy123222 <64176728+Spidy123222@users.noreply.github.com>

SternXD committed:
Merge pull request #550 from SideStore/Add-description-for-idle

Add description on what disable idle timeout toggle + change add to Siri text for accuracy

SternXD committed:
[skip ci] ci: Update stable.yml

This removes workflow_dispatch as it was making the builds show as SideStore `develop`

Signed-off-by: Stern <xsternent@gmail.com>

amsam0 committed:
[skip ci] Remove me from bug report assignees

Signed-off-by: naturecodevoid <44983869+naturecodevoid@users.noreply.github.com>

amsam0 committed:
[skip ci] Remove me from feature report assignees

Signed-off-by: naturecodevoid <44983869+naturecodevoid@users.noreply.github.com>

junepark678 committed:
change to self-hosted

junepark678 committed:
Update nightly.yml

Signed-off-by: June Park <rjp2030@outlook.com>

junepark678 committed:
feat(launch):add analytics notice

junepark678 committed:
ouch pushed selfhosted code

Signed-off-by: June Park <rjp2030@outlook.com>

junepark678 committed:
Update stable.yml

Signed-off-by: June Park <rjp2030@outlook.com>

junepark678 committed:
bugfix(launch):fix analytics notice

SternXD committed:
[skip ci] Adds ThatStella7922's Trusted Source

Signed-off-by: Stern <70122891+SternXD@users.noreply.github.com>

nythepegasus committed:
[skip ci] Fixed typo

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

nythepegasus committed:
Fix pairing file not resetting when embedded

nythepegasus committed:
Remove outdated/down servers for now

Eventually this could be dynamically pulled/inside SideStore
but extra DNS entries and cleaning can work for now

Spidy123222 committed:
Remove patreon exclusivity message for sources (#594)

polymo1 committed:
Add ani.wesbryie.com to Anisette list

Signed-off-by: Wes Bryie <wes@wesbryie.com>

SternXD committed:
Merge pull request #604 from wesbryiecom/develop

SternXD commented:
What's the status of this PR?

junepark678 commented:
stale

nythepegasus committed:
Actually fix embedded pairing file issue

stossy11 committed:
Add Stossy11 Anisette Server

SternXD committed:
Merge pull request #612 from stossy11/MyAnisette

Add Stossy11 Anisette Server

LifeDreamer24 commented:
anyone can pass me a build of this one? i simply cant find a way to enable jit within sidestore

WesleyBryie commented:
@junepark678 needs to force a build through GH actions, but its not up to date with the main branch anyways.

WesleyBryie commented:
FYI SideStore doesn't enable JIT on iOS 17+ itself.

LifeDreamer24 commented:
so there is no way to enable jit without pc?

WesleyBryie commented:
> so there is no way to enable jit without pc?
> 
> edit: on ios17

nope

stossy11 committed:
Add SideJITServer Support for Enabling JIT on iOS 17+ in app (#630)

SternXD commented:
@JoeMatt Have you tested this?

JoeMatt commented:
> @JoeMatt Have you tested this?

Nope, but the changes looked concise.

polymo1 committed:
add(readme): discord badge

bogotesr committed:
improve UX on intro popup

fixes that the url for the pairing file info was out of date

adds button for taking user to that page rather than just having a url in text

bogotesr committed:
Revert "improve UX on intro popup"

This reverts commit 2583c7f617ba7160e8891db88298a10b6d855dc6.

therealFoxster committed:
Rename "AltStore" to "SideStore"

SternXD committed:
Merge pull request #658 from therealFoxster/develop

Rename "AltStore" to "SideStore"

junepark678 committed:
merge AltStore 1.6.3, add dynamic anisette lists, merge SideJITServer integration

* Change error from Swift.Error to NSError

* Adds ResultOperation.localizedFailure

* Finish Riley's monster commit

3b38d725d7e8e45fb2c0cb465a3968828616c209
May the Gods have mercy on my soul.

* Fix format strings I broke

* Include "Enable JIT" errors in Error Log

* Fix minimuxer status checking

* [skip ci] Update the no wifi message to include VPN

* Opens Error Log when tapping ToastView

* Fixes Error Log context menu covering cell content

* Fixes Error Log context menu appearing while scrolling

* Fixes incorrect Search FAQ URL

* Fix Error Log showing UIAlertController on iOS 14+

* Fix Error Log not showing UIAlertController on iOS <=13

* Fix wrong color in AuthenticationViewController

* Fix typo

* Fixes logging non-AltServerErrors as AltServerError.underlyingError

* Limits quitting other AltStore/SideStore processes to database migrations

* Skips logging cancelled errors

* Replaces StoreApp.latestVersion with latestSupportedVersion + latestAvailableVersion

We now store the latest supported version as a relationship on StoreApp, rather than the latest available version. This allows us to reference the latest supported version in predicates and sort descriptors.

However, we kept the underlying Core Data property name the same to avoid extra migration.

* Conforms OperatingSystemVersion to Comparable

* Parses AppVersion.minOSVersion/maxOSVersion from source JSON

* Supports non-NSManagedObjects for @Managed properties

This allows us to use @Managed with properties that may or may not be NSManagedObjects at runtime (e.g. protocols). If they are, Managed will keep strong reference to context like before.

* Supports optional @Managed properties

* Conforms AppVersion to AppProtocol

* Verifies min/max OS version before downloading app + asks user to download older app version if necessary

* Improves error message when file does not exist at AppVersion.downloadURL

* Removes unnecessary StoreApp convenience properties

* Removes unnecessary StoreApp convenience properties as well as fix other issues

* Remove Settings bundle, add SwiftUI view instead

Fix refresh all shortcut intent

* Update AuthenticationOperation.swift

Signed-off-by: June Park <rjp2030@outlook.com>

* Fix build issues given by develop

* Add availability check to fix CI build(?)

* If it's gonna be that way...

---------

Signed-off-by: June Park <rjp2030@outlook.com>
Co-authored-by: nythepegasus <nythepegasus84@gmail.com>
Co-authored-by: Riley Testut <riley@rileytestut.com>
Co-authored-by: ny <me@nythepegas.us>

nythepegasus committed:
Bump SideStore version to 0.5.7

Signed-off-by: nythepegasus <nythepegasus84@gmail.com>

StephenDev0 committed:
Better AnisetteServerList.swift (NEEDS TESTING!!!!!!)

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

StephenDev0 committed:
Fixed Dark Mode for AnisetteServerList.swift

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

StephenDev0 committed:
Update AnisetteServerList.swift

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

SternXD committed:
Merge pull request #672 from 0-Blu/develop

UI Improvements for the Anisette Servers View.

StephenDev0 committed:
Added more feedback options.

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

StephenDev0 committed:
Update SettingsViewController.swift

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

StephenDev0 committed:
Update SettingsViewController.swift

Signed-off-by: Stephen <158498287+0-Blu@users.noreply.github.com>

SternXD committed:
Merge pull request #673 from 0-Blu/develop

Added more feedback options.

junepark678 committed:
Add nightly.link support

Pushing directly as this shouldn't mess anything other than CI up

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Make it so we don't flood altstore

Signed-off-by: June Park <me@pythonplayer123.dev>

stossy11 committed:
Fix SideJITServer Support for iOS 17+ (#674)

* FIx SideJITServer Support

* Fix SideJITServer Address

* Add Warning when Overwriting SideJITServer Address

* Fix Optional Value for SideJITServer URL

* Update SideJITServer Address Overwriting

* Fix Enabling JIT and Fix IP Address Loop

* Fix No WiFi or VPN! error when using SideJITServer

Spidy123222 commented:
I just rebased to see if it would work with more recent integrations and it doesn't work with Sidejitserver integration unfortunately.

stossy11 commented:
> I just rebased to see if it would work with more recent integrations and it doesn't work with Sidejitserver integration unfortunately.

I looked through the code quickly and found out its because it manually Debugs the app and doesn't use "AppManager.shared.enableJIT" func

junepark678 committed:
More descriptive errors

junepark678 committed:
Add nightly.link description

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Makes revoking optional (#675)

junepark678 committed:
more descriptive machine names for generated certs

junepark678 committed:
Delete AltStore.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Make app extensions optional across the board

junepark678 committed:
Add incremental builds

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Increase runtime performance

junepark678 committed:
[build] need to see if caching works

bogotesr committed:
Better pairing file info (#676)

makes the pop up give you a button to press for help rather than giving a plaintext url to a page that doesn't exist

nythepegasus committed:
Bump SideStore Version to 0.5.8

For this current nightly track of commits until we "officially" release it

Signed-off-by: nythepegasus <me@nythepegas.us>

junepark678 committed:
Make caches not have workflow name

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
[skip ci] add caches to beta

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
[skip ci] add caches to stable.yml

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Fix certificate issues

junepark678 committed:
Remove cache clear

We don't need to do this anymore

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Make app extension popup less annoying by not showing when refreshing (it doesn't do anything anyway)

junepark678 committed:
[no ci] Update pr.yml to cache builds

Signed-off-by: June Park <me@pythonplayer123.dev>

junepark678 committed:
Selective app extension removal (#677)

junepark678 committed:
add entitlements

junepark678 committed:
[skip ci] Update bug_report.yml

Signed-off-by: June Park <me@pythonplayer123.dev>

nythepegasus committed:
Attempt to fix sharing ipa files

Yeah, yeah, I shouldn't commit to develop, I have a hunch though

Signed-off-by: nythepegasus <me@nythepegas.us>

polymo1 committed:
legal: developer's certificate of origin 1.1 (#686)

nythepegasus committed:
[skipci] Bump to 0.5.9 for new nightly cycle

Signed-off-by: nythepegasus <me@nythepegas.us>

SternXD committed:
(fix:) Update Patreon Secret and links

Signed-off-by: Stern <stern@sidestore.io>

nythepegasus committed:
Add bad app limit toggle implementation

nythepegasus committed:
Fix up this so it actually sets the value

nythepegasus committed:
Fix these actions to point correctly

nythepegasus committed:
Add a max limit

nythepegasus committed:
Add a few more checks for app limits

nythepegasus committed:
Remove this

Should fix remove app extensions

Signed-off-by: nythepegasus <me@nythepegas.us>

nythepegasus committed:
Try to fix refreshing and removing extensions

Silly fix

Signed-off-by: nythepegasus <me@nythepegas.us>

nythepegasus committed:
Update GH Actions to fix deprecation warnings

nythepegasus committed:
Actually fix deprecation warnings

moonsn committed:
feat: show custom anisette server list if set (#717)

nythepegasus committed:
Fix detection for SparseRestore patch

nythepegasus committed:
Clean/fix this implementation

nythepegasus committed:
Attempt to make the bypass more sane

nythepegasus committed:
More attempts at fixing app limit issues

mahee96 committed:
Fix for Refresh Operation causing renewal/install of provisioning profiles for removed app extensions (#727)

* Fix AppExtensions not being updated on Disk after db is updated in InstallAppOperation

* refresh-Extensions: Added check to ensure extensions in DB and DISK matches if not then cancel current refresh request

mahee96 committed:
- Fix: DatabaseManager.migrateDatabaseToAppGroupIfNeeded() src and dest for replaceAt() cannot be same

mahee96 committed:
xcode: Debug dylib is still broken in iOS 18 causing "No Entry Point Found. checked(null)"

mahee96 committed:
Fix-regression: restored update-check logic(2a392dd) in InstalledApp.swift which was dropped in (1713fcc) (#728)

junepark678 committed:
Merge pull request #729 from mahee96/sidestore-crash-xcode

xcode: Debug dylib support is still broken in iOS 18 causing "No Entry Point Found. checked(null)"

mahee96 committed:
refresh-app: enhanced error logging when app extension validation check fails (#730)

refresh-app: enhanced error logging when app exenstion validation check fails

SternXD committed:
chore: Add checks to fetch-prebuilt script.

This adds a check that if for example wget isn't installed it'll install from brew via brew install.

Signed-off-by: Stern <stern@sidestore.io>

SternXD committed:
chore: We need to add Homebrew to our path or Xcode will not find it.

Signed-off-by: Stern <stern@sidestore.io>

mahee96 committed:
diagnostics: improved error logging for OperationError.invalidParameters (#736)

mahee96 committed:
restore changes dropped from altstore by 1713fcc

SternXD committed:
Merge pull request #738 from mahee96/develop

-[cleanup]: restore changes dropped from altstore-1.6.3 merge by 1713fcc

TheMasterOfMike committed:
Add missing = sign (#739)

mahee96 committed:
-[bug-fix]: UI Api invocation needs to be on UI Thread

mahee96 committed:
-[cleanup]: added guard check to remove file only if it exists

mahee96 committed:
-[diagnostics]: Added diagnostics for RefreshAppOperation failure

mahee96 committed:
Partial-Fix for #723: Error on refresh operation must be set on self.context instead of group.context (#742)

mahee96 committed:
Background and shortcut refresh fix - remove only excess extensions from new package during install (#743)

mahee96 committed:
Widgets-Fix: Ported Widget compatibility for ios 18+ from altstore  (#746)

kaoneko committed:
Update message about Disable Idle Timeout setting (#747)

JoeMatt committed:
Change gitignore, add package.resolve (#748)

SternXD committed:
(chore:) Update Twitter Link

Signed-off-by: Stern <stern@sidestore.io>

StephenDev0 committed:
Twitter Updates & Email Support (#762)

mahee96 committed:
-[xcode]: revert enabling debug dylib support for widgets

mahee96 committed:
Fix: update sign-in screen toast text color to be primary color

SternXD committed:
Merge pull request #770 from mahee96/develop

mahee96 committed:
-[Feature]: client-side: Anisette server fallback impl when current server is unreachable (#773)

mahee96 committed:
- Fix: refresh(): app was being accessed after de-alloc causing EXC_BREAKPOINT

mahee96 committed:
- Fix: Anisette-server-fallback: Try currently selected server first

mahee96 committed:
- Fix: AnisetteServers: refresh server request shouldn't use local cache to get proper updates

mahee96 committed:
- Show Toast regarding server switch due to fallback if UI context is available

nythepegasus committed:
fix: Refreshing via Xcode would cause a crash here

mahee96 committed:
Added launch arguments for coredata debugging and updated Scheme from AltStore to SideStore (#785)

mahee96 committed:
- renamed libem_proxy.a to libem_proxy-ios.a for iOS builds

mahee96 committed:
Updated LIB_FILE_NAME used in minimuxer shell script to link the binary

mahee96 committed:
updated fetch-prebuilt.sh to reflect libem_proxy-ios.a renaming

mahee96 committed:
[fetch-prebuilt.sh] - retain checked-in files from minimuxer repo (assuming minimuxer repo was cloned)

Spidy123222 commented:
This should be fine long as it detects for iOS 17+ versions with the sidejitserver mode disabled

michaelmelita1 commented:
Does anyone want to try this with the latest JIT implementation
